desc/backend.py

```diff
@@ -550,8 +610,16 @@ def vmap(fun, out_axes=0):
             Vectorized version of fun.
 
         """
+        if in_axes != 0:
+            raise NotImplementedError(
+                f"Backend for numpy vmap for in_axes={in_axes} not implemented yet."
+            )
 
         def fun_vmap(fun_inputs):
+            if isinstance(fun_inputs, tuple):
+                raise NotImplementedError(
```

Comment:
This needs to be implemented if we want DESC to work on the numpy on backend without jax.